library/src/scala/quoted/Quotes.scala

```diff
@@ -3649,6 +3670,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
         /** Type member directly declared in the class */
         def typeMembers: List[Symbol]
 
```

Comment:
I see there some inconsistency as `methodMembers` and `fieldMembers` return both declared and inherited members and `typeMembers` returns only declared members.